runtime/src/lib.rs

```diff
@@ -54,7 +54,7 @@ use sr_staking_primitives::SessionIndex;
 use srml_support::{
 	parameter_types, construct_runtime, traits::{SplitTwoWays, Currency}
 };
-use im_online::AuthorityId as ImOnlineId;
+use im_online::sr25519::{AuthorityId as ImOnlineId};
```

Comment:
we use this one in node runtime in substrate though I don't know which one we want here

Reply:
might help 

https://github.com/paritytech/substrate/pull/3500#discussion_r318454237